site/content/docs/heartbeat-checks/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+
+This guide gives you all the info needed to get started with Checkly Heartbeat checks.
+
+## What is a Heartbeat check?
+
+A heartbeat check is a passive check type that expects pings from an external source, such as a scheduled job on a server, at a defined interval. A ping is a HTTP requests to a given endpoint URL. 
+When a ping is not received on time, the check will trigger any configured alerts.
+
+Use heartbeat checks to monitor backup jobs, data imports, and other recurring jobs or scripts.
+
+Here is an example of how to have a Heroku job send a ping to a Checkly Heartbeat check.
+{{< tabs "Heroku example" >}}
+{{< tab "BASH" >}}
+```BASH
+curl -m 5 --retry 3 https://api.checklyhq.com/heartbeats/ping/bcd964a7-6f15-49a5-bac1-4be8059670ec;
+```
+{{< /tab >}}
+{{< /tabs >}}
+Note the use of the retry option. We recommend always using retries when available to avoid false alarms due to temporary network issues or other disruptions.
+
+## Creating a heartbeat check
+
+To create a new Heartbeat check, click the + icon on the sidebar & select **Heartbeat check**.
+
+Creating a heartbeat check is quick and easy; the check requires a name and the period and grace settings defined. These can all be changed later on. Optionally, you can customize tags or [alert settings](/docs/alerting).
+
+Once you have created your check, the service or host you want to monitor needs to be configured to send a request to the ping URL. When creating or editing the check you can find code examples for how to send requests using Javascript, Python or in Bash in the quickstart section.
+
+## Check breakdown
+
+### Name and tag
+The check name is used for identifying the check in the list of heartbeat checks and in alert messages. Using a clear and meaningful name will help team members identify the check and can help reduce the reaction time when alerts are triggered. 
+Tags are used to create meaningful distinctions between check groups, making it easy to filter out selections of checks. 
+
+### Ping URL
+The URL on which the check is listening for pings. The job or task monitored should make an HTTP request to the ping URL once per the period configured.
+
+### Period and Grace
+**Period** defines how often you expect a ping to the ping URL. 
+
+**Grace** is the time Checkly will wait before triggering any alerts when a ping does not arrive within the set period. E.g., if you have a check that expects a ping every 60 minutes, with a grace of 10 minutes, no alarms would trigger until 70 minutes after the latest ping.
+
+Use grace to compensate for variance in your jobs.
+
+### Timer
+The check timer starts when it receives its first ping and will reset after each ping. 
+If you have a check that expects a ping every 60 minutes starting at 09:30, if it receives a ping at 10:00, it will reset the timer to expect a ping before 11:00. If the check does not receive a ping before 11:00 plus any configured grace period it will trigger any configured alerts.
```

Comment:
Maybe we can add a diagram here with this information?

```suggestion
If you have a check that expects a ping every 60 minutes starting at 09:30, and it receives a ping at 10:00, it will reset the timer to expect a ping before 11:00. If the check does not receive a ping before 11:00 plus any configured grace period it will trigger any configured alerts.
```

Reply:
@miliberlin : Yup, good idea. I'll see if I can come up with a good visualization. Any ideas?

Reply:
Miro has some nice templates. Maybe [this one](https://miro.com/app/dashboard/?tpTemplate=timeline-builder&isCustom=false&share_link_id=347710956168)?